integration-cli/docker_cli_push_test.go

```diff
@@ -577,4 +577,6 @@ func (s *DockerRegistryAuthTokenSuite) TestPushMisconfiguredTokenServiceResponse
 	dockerCmd(c, "tag", "busybox", repoName)
 	out, _, err := dockerCmdWithError("push", repoName)
 	c.Assert(err, check.NotNil, check.Commentf(out))
+	split := strings.Split(out, "\n")
+	c.Assert(split[len(split)-2], check.Equals, "unauthorized: authentication required")
```

Comment:
alright, but before this distribution vendor I got just <nil> so 401 wasn't handled that way - I guess it's fine though right?